src/main/java/com/github/vkorobkov/jfixtures/config/Config.java

```diff
@@ -17,6 +18,11 @@ public Config(String fixturesRoot) {
         return fixturesConf.digValue("refs", table, column);
     }
 
+    public boolean shouldAutoGeneratePk(String tableName) {
+        Optional<Boolean> autoGeneratePkFlag = fixturesConf.digValue(AUTO_GENERATE_PK, tableName);
+        return !autoGeneratePkFlag.isPresent() || autoGeneratePkFlag.get();
```

Comment:
Use `java.util.Optional#orElse(T other)` instead of this complex condition. That should work:
```
return fixturesConf.digValue(AUTO_GENERATE_PK, tableName).orElse(true);
```